string Protocol
int32 pwmDir
int32 pwmProp
float32 Vcons
float32 Kp
float32 Ki

